src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -23,9 +24,17 @@ namespace Details
 // Magic numbers generated by the script in
 // https://stackoverflow.com/questions/1024754/how-to-compute-a-3d-morton-number-interleave-the-bits-of-3-ints/18528775#18528775
 
+template <int N>
+KOKKOS_INLINE_FUNCTION unsigned int expandBitsBy(unsigned int)
+{
+  static_assert(0 < N && N < 6,
+                "expandBitsBy can only be used with values 1-5");
+  return UINT_MAX;
```

Comment:
```suggestion
  return 0;
```
so we don't rely on `<climits>` being transitively included

Reply:
I thought about 0, but decided that noticing wrong 0 is much harder when debugging compared to noticing large numbers.

Reply:
Just call `Kokkos::abort` before returning

Reply:
Done.